### ISSUE TITLE
componentWillMount has been renamed and not recommended for use.

### ISSUE THREAD

**Issue by vic3lpl** [maggialejandro/react-native-calendario#104]
any update to solve this warning "componentWillMount has been renamed and not recommended for use." ?

**Comment by Technaur** [comment]
I am also getting the same warning. On version 0.4.0 of `react-native-calendario` and version 0.61.3 of `react-native`.

```
setUpDeveloperTools.js:73 Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Calendar
```

**Comment by maggialejandro** [comment]
Fixed in 0.5.0
I'm closing this for now
